pkg/gen/python-templates/yaml.py.mustache

```diff
@@ -0,0 +1,132 @@
+# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import json
+from copy import deepcopy
+from typing import Optional
+
+import pulumi
+import pulumi.runtime
+import requests
+import yaml
+{{#Groups}}
+{{#Versions}}
+from pulumi_kubernetes.{{Group}}.{{Version}} import *
+{{/Versions}}
+{{/Groups}}
+
+from . import tables
+
+
+class ConfigFile(pulumi.ComponentResource):
+    """
+    ConfigFile creates a set of Kubernetes resources from a Kubernetes YAML file. If `config.name`
+    is not specified, `ConfigFile` assumes the argument `name` is the filename.
```

Comment:
Can you add Sphinx parameter docs here? Like this:

https://github.com/pulumi/pulumi-aws/blob/ff6e012c8cea4a9f13bdaf7d00a5f35505b198af/sdk/python/pulumi_aws/provider.py#L19-L20

```
    :param str name A name for a resource
```
etc.